### PR TITLE
fetch-lib

### PR DESCRIPTION
**Pull request by sed-i**
## Issue
Exception on ingress removal.

## Solution
Fetch lib.

### CODE REVIEW

File: lib/charms/traefik_route_k8s/v0/traefik_route.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def __init__(self, *args):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 3
+LIBPATCH = 2
```
Comment (Abuelodelanada) [inline lib/charms/traefik_route_k8s/v0/traefik_route.py:91]: Are we downgrading the lib?

**Comment by sed-i** [comment]
charmcarft says:
```
Library charms.grafana_auth.v0.grafana_auth has local changes, cannot be updated.
Library charms.grafana_k8s.v0.grafana_auth was already up to date in version 0.1.
```
charm code only uses
```
from charms.grafana_auth.v0.grafana_auth import AuthRequirer, AuthRequirerCharmEvents
```

Both files have the same LIBID.

Any cleanup to do here? @rbarry82 @saltiyazan

File: lib/charms/traefik_route_k8s/v0/traefik_route.py (hunk 1 above)
Comment (sed-i) [reply lib/charms/traefik_route_k8s/v0/traefik_route.py:91]: I think it was incremented without publishing.

**Review by saltiyazan — APPROVED** [review]

**Comment by sed-i** [comment]
CI fails for reasons documented in #137. Merging.

**Pull request merged**